src/InvoiceBundle/Action/Create.php

```diff
@@ -41,8 +41,12 @@ public function __construct(FormHandler $handler, ClientRepository $clientReposi
 
     public function __invoke(Request $request, Client $client = null)
     {
-        if (!$this->clientRepository->getTotalClients()) {
+        $totalClientsCount = $this->clientRepository->getTotalClients();
+        if (!$totalClientsCount) {
             return new Template('@SolidInvoiceInvoice/Default/empty_clients.html.twig');
+        } elseif (1 === $totalClientsCount && is_null($client)){
+            $clients = $this->clientRepository->findAll();
+            $client = reset($clients);
```

Comment:
This can change to `$client = $this->clientRepository->findOneBy([]);` in order to only get the first client and not have to worry about getting an array back